pkg/controller/plan/adapter/ovirt/builder.go

```diff
@@ -391,6 +449,26 @@ func (r *Builder) mapCPU(vm *model.Workload, object *cnv.VirtualMachineSpec) {
 	if vm.CpuPinningPolicy == model.Dedicated {
 		object.Template.Spec.Domain.CPU.DedicatedCPUPlacement = true
 	}
+	var fullCpu string
+	if vm.CustomCpuModel != "" {
+		fullCpu = vm.CustomCpuModel
+	} else if r.Plan.Spec.PerserveClusterCPUModel {
+		fullCpu = clusterCpuMap[vm.Cluster.CPU.Type]
+	} else {
+		return
+	}
+	object.Template.Spec.Domain.CPU.Model = cpuMap[fullCpu]
+	cpuFlags := strings.Split(fullCpu, ",")
+	for _, val := range cpuFlags[1:] {
+		var flag string
+		flag, found := strings.CutPrefix(val, "+")
+		if !found {
+			flag, _ = strings.CutPrefix(val, "-")
+			object.Template.Spec.Domain.CPU.Features = append(object.Template.Spec.Domain.CPU.Features, cnv.CPUFeature{Name: flag, Policy: "disable"})
+		} else {
+			object.Template.Spec.Domain.CPU.Features = append(object.Template.Spec.Domain.CPU.Features, cnv.CPUFeature{Name: flag, Policy: "require"})
+		}
```

Comment:
but we can also have features without '+' or '-', right?

Reply:
i didn't see that in ovirt, but it is possible in theory. i saw that the default case is `+`, therefore it defaults here as well.

Reply:
ok, I'd prefer to delegate this decision to libvirt (to determine the default) but if you want to set it here, it's also fine
but are you sure this code default to 'require'? from a brief look, it seems that the logic is: "if the prefix is `+`, set the policy to `required`, otherwise assume the prefix is `-`.."

Reply:
in [ovirt](https://github.com/oVirt/ovirt-engine/blob/master/backend/manager/modules/vdsbroker/src/main/java/org/ovirt/engine/core/vdsbroker/builder/vminfo/LibvirtVmXmlBuilder.java#L503-L518) seem it is delegate to libvirt. in libvirt: `Since 0.8.5 the policy attribute can be omitted and will default to require.` [libvirt doc](https://libvirt.org/formatdomain.html#cpu-model-and-topology). i think we can drop it and to like ovirt (delegate).